<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEquipment</class>
 <widget class="QDialog" name="AddEquipment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>افزودن دستگاه</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLineEdit" name="locationEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="serialCode">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>90000</number>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>ثبت دستگاه</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPlainTextEdit" name="equipment_note">
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="typeComboBox">
     <item>
      <property name="text">
       <string>PU1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PU2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PU3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PU4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PU5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PU6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PU7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PU8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DESMA</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>شماره سریال:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>محل استقرار:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>تیپ دستگاه:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>توضیحات</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
